src/core/core.tsx

```diff
@@ -179,24 +177,6 @@ export const blockExplorerTransactionLink = (
   return chainIdToNetwork[chainID].getExplorerTransactionLink(transactionHash);
 };
 
-// pass in [JSON].abi
```

Comment:
Why is this being deleted?

Reply:
This is not being used since we're not loading up any contracts